dynamodb/client_test.go

```diff
@@ -0,0 +1,220 @@
+package dynamodbclient
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/stretchr/testify/assert"
+	"reflect"
+	"testing"
+)
+
+const (
+	UUID         = "4f50b156-6c50-4693-b835-02f70d3f3bc0"
+	DDB_TABLE    = "upp-concordance-store-semantic"
+	AWS_REGION   = "eu-west-1"
+	DDB_ENDPOINT = "http://localhost:8000"
+)
+
+var goodModel = Model{
+	UUID:         UUID,
+	ConcordedIds: []string{"7c4b3931-361f-4ea4-b694-75d1630d7746", "1e5c86f8-3f38-4b6b-97ce-f75489ac3113"},
+}
+
+var db *dynamodb.DynamoDB
+var c DynamoDbClientImpl
+
+var DescribeTableParams = &dynamodb.DescribeTableInput{TableName: aws.String(DDB_TABLE)}
+
+func init() {
```

Comment:
If these tests rely on a local instance of DynamoDB running, can we implement skipping them using the -short flag so that the other tests can be run easier.